src/main/resources/sledgefile.yaml

```diff
@@ -1,12 +1,12 @@
 # Sledgefile example
 
-artifactsPathPrefix: artifacts/
+artifactsPathPrefix: packages/
 
 deploymentDefs:
-  - name: "local"
+  - name: "local-author"
     artifacts:
-      - my-package-1.0.zip
-      - my-package2-1.0.zip
+      - ktbe-newweb-aem-apps-0.0.5-SNAPSHOT.zip
```

Comment:
we should use here generic package names
Currently I always copied a version into the target/build directory and adapted it to do the tests...

Reply:
true :) it should be removed and file should be renamed!